### PR TITLE
Capturing technical nuances in Slim CI

### PR DESCRIPTION
**Pull request by runleonarun**
## What are you changing in this pull request and why?

Capturing the technical details @joellabes [suggested in a previous fix](https://github.com/dbt-labs/docs.getdbt.com/pull/2800/files) to improve this page. 

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/deploy/cloud-ci-job.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ dbt Cloud might not drop the temporary schema from your data warehouse if your p
 
 ### Configuring a webhook
```
Comment (runleonarun) [inline website/docs/docs/deploy/cloud-ci-job.md]:
```suggestion
### Configure a job trigger
```

File: website/docs/docs/deploy/cloud-ci-job.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ dbt Cloud might not drop the temporary schema from your data warehouse if your p
...
 
-If you want dbt Cloud to run the job whenever a pull request or commit is made, rather than on a schedule, you can set up a webhook. 
+If you want dbt Cloud to run the job whenever a pull request or commit is made, you can set up a webhook.
```
Comment (runleonarun) [inline website/docs/docs/deploy/cloud-ci-job.md]:
```suggestion
If you want dbt Cloud to run the job whenever a pull request or commit is made, you can set up a job trigger. 
```

File: website/docs/docs/deploy/cloud-ci-job.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ description: "You can enable continuous integration (CI) to test every single ch
 ---
 
 
-dbt Cloud makes it easy to test every single code change you make prior to deploying that new logic into production. Once you've connected your [GitHub account](/docs/collaborate/git/connect-github), [GitLab account](/docs/collaborate/git/connect-gitlab), or [Azure DevOps account](/docs/collaborate/git/connect-azure-devops), you can configure continuous integration (CI) jobs to run when someone opens a new pull request in your dbt repository by setting up a webhook. For more information, refer to [Configuring a webhook](#configuring-a-webhook).
+dbt Cloud makes it easy to test every single code change you make prior to deploying that new logic into production. Once you've connected your [GitHub account](/docs/collaborate/git/connect-github), [GitLab account](/docs/collaborate/git/connect-gitlab), or [Azure DevOps account](/docs/collaborate/git/connect-azure-devops), you can configure continuous integration (CI) jobs to run when someone opens a new pull request in your dbt repository. For more information, refer to [Configuring a webhook](#configuring-a-webhook).
```
Comment (runleonarun) [inline website/docs/docs/deploy/cloud-ci-job.md]:
```suggestion
dbt Cloud makes it easy to test every single code change you make prior to deploying that new logic into production. Once you've connected your [GitHub account](/docs/collaborate/git/connect-github), [GitLab account](/docs/collaborate/git/connect-gitlab), or [Azure DevOps account](/docs/collaborate/git/connect-azure-devops), you can configure continuous integration (CI) jobs to run when someone opens a new pull request in your dbt repository. For more information, refer to [Configuring a job trigger](#configuring-a-job-trigger).
```

File: website/docs/docs/deploy/cloud-ci-job.md [hunk 4]
```diff
@@ -24,17 +24,17 @@ If you previously configured your dbt project by providing a generic git URL tha
...
 
 <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/using_ci_dbt_cloud.png" title="Viewing the temporary schema name for a run triggered by a PR"/>
 
 After completing the dbt run, dbt Cloud will update the pull request in GitHub, GitLab, or Azure DevOps with a status message indicating the results of the run. The status message will state whether the models and tests ran successfully or not. You can enable a setting in your git provider that makes "successful pull request checks" a requirement to merge code. And finally, once the pull request is closed or merged, dbt Cloud will delete the temporary schema from your <Term id="data-warehouse" />.
 
 dbt Cloud might not drop the temporary schema from your data warehouse if your project has database / schema customization using the [`generate_database_name`](/docs/build/custom-databases#generate_database_name) / [`generate_schema_name`](/docs/build/custom-schemas#how-does-dbt-generate-a-models-schema-name) macros. For more information, refer to [Temp PR schema limitations](/docs/deploy/cloud-ci-job#temp-pr-schema-limitations).
 
-### Configuring a webhook
+### Configure a job trigger
```
Comment (runleonarun) [inline website/docs/docs/deploy/cloud-ci-job.md]: @joellabes I tried to clarify that this is not the newly released webhooks, but a job trigger as seen here (the tab is actually also called "webhook"): 
Screenshot 2023-02-02 at 1 03 59 PM

Comment (joellabes) [reply website/docs/docs/deploy/cloud-ci-job.md]: Yeah I think you’ve doen a good job threading the needle here!

Comment (joellabes) [reply website/docs/docs/deploy/cloud-ci-job.md]: I've [asked in the webhooks channel](https://dbt-labs.slack.com/archives/C04KR8M25AM/p1675375172516919) if we should rebrand this page

**Review by joellabes — DISMISSED** [review]
lgtm!

File: website/docs/docs/deploy/cloud-ci-job.md [hunk 5]
```diff
@@ -5,7 +5,7 @@ description: "You can enable continuous integration (CI) to test every single ch
 ---
 
 
-dbt Cloud makes it easy to test every single code change you make prior to deploying that new logic into production. Once you've connected your [GitHub account](/docs/collaborate/git/connect-github), [GitLab account](/docs/collaborate/git/connect-gitlab), or [Azure DevOps account](/docs/collaborate/git/connect-azure-devops), you can configure continuous integration (CI) jobs to run when someone opens a new pull request in your dbt repository by setting up a webhook. For more information, refer to [Configuring a webhook](#configuring-a-webhook).
+dbt Cloud makes it easy to test every single code change you make prior to deploying that new logic into production. Once you've connected your [GitHub account](/docs/collaborate/git/connect-github), [GitLab account](/docs/collaborate/git/connect-gitlab), or [Azure DevOps account](/docs/collaborate/git/connect-azure-devops), you can configure continuous integration (CI) jobs to run when someone opens a new pull request in your dbt repository. For more information, refer to [Configuring a job trigger](#configuring-a-job-trigger).
```
Comment (runleonarun) [inline website/docs/docs/deploy/cloud-ci-job.md]:
```suggestion
dbt Cloud makes it easy to test every single code change you make prior to deploying that new logic into production. Once you've connected your [GitHub account](/docs/collaborate/git/connect-github), [GitLab account](/docs/collaborate/git/connect-gitlab), or [Azure DevOps account](/docs/collaborate/git/connect-azure-devops), you can configure continuous integration (CI) jobs to run when someone opens a new pull request in your dbt repository. For more information, refer to [Configure a job trigger](#configure-a-job-trigger).
```

File: website/docs/docs/deploy/cloud-ci-job.md [hunk 6]
```diff
@@ -24,17 +24,17 @@ If you previously configured your dbt project by providing a generic git URL tha
 
 When you set up a dbt Cloud CI job with a [webhook configured](#configuring-a-webhook), dbt Cloud will listen for webhooks from GitHub, GitLab, or Azure DevOps indicating that a new pull request has been opened or updated with new commits. When one of these webhooks is received, dbt Cloud will enqueue a new run of the CI job.
```
Comment (runleonarun) [inline website/docs/docs/deploy/cloud-ci-job.md]:
```suggestion
When you set up a dbt Cloud CI job with a [job trigger](#configure-a-job-trigger), dbt Cloud will listen for webhooks from GitHub, GitLab, or Azure DevOps indicating that a new pull request has been opened or updated with new commits. When one of these webhooks is received, dbt Cloud will enqueue a new run of the CI job. 
```

**Review by joellabes — DISMISSED** [review]

File: website/docs/docs/deploy/cloud-ci-job.md (hunk 4 above)
Comment (runleonarun) [reply website/docs/docs/deploy/cloud-ci-job.md]: This new UI is now part of this PR thanks to @thorn14

File: website/docs/docs/deploy/cloud-ci-job.md [hunk 7]
```diff
@@ -22,27 +22,27 @@ If you previously configured your dbt project by providing a generic git URL tha
...
 
 <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/using_ci_dbt_cloud.png" title="Viewing the temporary schema name for a run triggered by a PR"/>
 
 After completing the dbt run, dbt Cloud will update the pull request in GitHub, GitLab, or Azure DevOps with a status message indicating the results of the run. The status message will state whether the models and tests ran successfully or not. You can enable a setting in your git provider that makes "successful pull request checks" a requirement to merge code. And finally, once the pull request is closed or merged, dbt Cloud will delete the temporary schema from your <Term id="data-warehouse" />.
 
 dbt Cloud might not drop the temporary schema from your data warehouse if your project has database / schema customization using the [`generate_database_name`](/docs/build/custom-databases#generate_database_name) / [`generate_schema_name`](/docs/build/custom-schemas#how-does-dbt-generate-a-models-schema-name) macros. For more information, refer to [Temp PR schema limitations](/docs/deploy/cloud-ci-job#temp-pr-schema-limitations).
```
Comment (runleonarun) [inline website/docs/docs/deploy/cloud-ci-job.md]:
```suggestion
dbt Cloud might not drop the temporary schema from your data warehouse if your project has database or schema customization using the [`generate_database_name`](/docs/build/custom-databases#generate_database_name) or [`generate_schema_name`](/docs/build/custom-schemas#how-does-dbt-generate-a-models-schema-name) macros. For more information, refer to [Temp PR schema limitations](/docs/deploy/cloud-ci-job#temp-pr-schema-limitations).
```

File: website/docs/docs/deploy/cloud-ci-job.md [hunk 8]
```diff
@@ -22,27 +22,27 @@ If you previously configured your dbt project by providing a generic git URL tha
...
 
-dbt Cloud might not drop the temporary schema from your data warehouse if your project has database / schema customization using the [`generate_database_name`](/docs/build/custom-databases#generate_database_name) / [`generate_schema_name`](/docs/build/custom-schemas#how-does-dbt-generate-a-models-schema-name) macros. For more information, refer to [Temp PR schema limitations](/docs/deploy/cloud-ci-job#temp-pr-schema-limitations).
+dbt Cloud might not drop the temporary schema from your data warehouse if your project has database or schema customization using the [`generate_database_name`](/docs/build/custom-databases#generate_database_name) or [`generate_schema_name`](/docs/build/custom-schemas#how-does-dbt-generate-a-models-schema-name) macros. For more information, refer to [Temp PR schema limitations](/docs/deploy/cloud-ci-job#temp-pr-schema-limitations).
 
-### Configuring a webhook
+### Configure a job trigger
 
-If you want dbt Cloud to run the job whenever a pull request or commit is made, rather than on a schedule, you can set up a webhook. 
+If you want dbt Cloud to run the job whenever a pull request or commit is made, you can set up a job trigger.
```
Comment (runleonarun) [inline website/docs/docs/deploy/cloud-ci-job.md]:
```suggestion
If you want dbt Cloud to run the job whenever a pull request or commit is made, you can set up continuous integration (CI) for the job. 
```

File: website/docs/docs/deploy/cloud-ci-job.md [hunk 9]
```diff
@@ -22,27 +22,27 @@ If you previously configured your dbt project by providing a generic git URL tha
...
+dbt Cloud might not drop the temporary schema from your data warehouse if your project has database or schema customization using the [`generate_database_name`](/docs/build/custom-databases#generate_database_name) or [`generate_schema_name`](/docs/build/custom-schemas#how-does-dbt-generate-a-models-schema-name) macros. For more information, refer to [Temp PR schema limitations](/docs/deploy/cloud-ci-job#temp-pr-schema-limitations).
 
-### Configuring a webhook
+### Configure a job trigger
 
-If you want dbt Cloud to run the job whenever a pull request or commit is made, rather than on a schedule, you can set up a webhook. 
+If you want dbt Cloud to run the job whenever a pull request or commit is made, you can set up a job trigger. 
 
 To set up a webhook:
```
Comment (runleonarun) [inline website/docs/docs/deploy/cloud-ci-job.md]:
```suggestion
To set up CI:
```

File: website/docs/docs/deploy/cloud-ci-job.md [hunk 10]
```diff
@@ -22,27 +22,27 @@ If you previously configured your dbt project by providing a generic git URL tha
...
 
-dbt Cloud might not drop the temporary schema from your data warehouse if your project has database / schema customization using the [`generate_database_name`](/docs/build/custom-databases#generate_database_name) / [`generate_schema_name`](/docs/build/custom-schemas#how-does-dbt-generate-a-models-schema-name) macros. For more information, refer to [Temp PR schema limitations](/docs/deploy/cloud-ci-job#temp-pr-schema-limitations).
+dbt Cloud might not drop the temporary schema from your data warehouse if your project has database or schema customization using the [`generate_database_name`](/docs/build/custom-databases#generate_database_name) or [`generate_schema_name`](/docs/build/custom-schemas#how-does-dbt-generate-a-models-schema-name) macros. For more information, refer to [Temp PR schema limitations](/docs/deploy/cloud-ci-job#temp-pr-schema-limitations).
 
-### Configuring a webhook
+### Configure a job trigger
```
Comment (runleonarun) [inline website/docs/docs/deploy/cloud-ci-job.md]:
```suggestion
### Configure CI for a job
```

File: website/docs/docs/deploy/cloud-ci-job.md [hunk 11]
```diff
@@ -22,27 +22,27 @@ If you previously configured your dbt project by providing a generic git URL tha
 
 ## Configuring continuous integration in dbt Cloud
 
-When you set up a dbt Cloud CI job with a [webhook configured](#configuring-a-webhook), dbt Cloud will listen for webhooks from GitHub, GitLab, or Azure DevOps indicating that a new pull request has been opened or updated with new commits. When one of these webhooks is received, dbt Cloud will enqueue a new run of the CI job. 
+When you set up a dbt Cloud CI job with a [job trigger](#configure-a-job-trigger), dbt Cloud will listen for webhooks from GitHub, GitLab, or Azure DevOps indicating that a new pull request has been opened or updated with new commits. When one of these webhooks is received, dbt Cloud will enqueue a new run of the CI job.
```
Comment (runleonarun) [inline website/docs/docs/deploy/cloud-ci-job.md]:
```suggestion
When you [set up CI for a job](#configure-ci-for-a-job), dbt Cloud will listen for webhooks from GitHub, GitLab, or Azure DevOps indicating that a new pull request has been opened or updated with new commits. When one of these webhooks is received, dbt Cloud will enqueue a new run of the CI job. 
```

File: website/docs/docs/deploy/cloud-ci-job.md [hunk 12]
```diff
@@ -5,7 +5,7 @@ description: "You can enable continuous integration (CI) to test every single ch
 ---
 
 
-dbt Cloud makes it easy to test every single code change you make prior to deploying that new logic into production. Once you've connected your [GitHub account](/docs/collaborate/git/connect-github), [GitLab account](/docs/collaborate/git/connect-gitlab), or [Azure DevOps account](/docs/collaborate/git/connect-azure-devops), you can configure continuous integration (CI) jobs to run when someone opens a new pull request in your dbt repository by setting up a webhook. For more information, refer to [Configuring a webhook](#configuring-a-webhook).
+dbt Cloud makes it easy to test every single code change you make prior to deploying that new logic into production. Once you've connected your [GitHub account](/docs/collaborate/git/connect-github), [GitLab account](/docs/collaborate/git/connect-gitlab), or [Azure DevOps account](/docs/collaborate/git/connect-azure-devops), you can configure continuous integration (CI) jobs to run when someone opens a new pull request in your dbt repository. For more information, refer to [Configure a job trigger](#configure-a-job-trigger).
```
Comment (runleonarun) [inline website/docs/docs/deploy/cloud-ci-job.md]:
```suggestion
dbt Cloud makes it easy to test every single code change you make prior to deploying that new logic into production. Once you've connected your [GitHub account](/docs/collaborate/git/connect-github), [GitLab account](/docs/collaborate/git/connect-gitlab), or [Azure DevOps account](/docs/collaborate/git/connect-azure-devops), you can configure continuous integration (CI) jobs to run when someone opens a new pull request in your dbt repository. For more information, refer to [Configure CI for a job](#configure-ci-for-a-job).
```

File: website/docs/docs/deploy/cloud-ci-job.md [hunk 13]
```diff
@@ -22,27 +22,27 @@ If you previously configured your dbt project by providing a generic git URL tha
...
-### Configuring a webhook
+### Configure CI for a job
 
-If you want dbt Cloud to run the job whenever a pull request or commit is made, rather than on a schedule, you can set up a webhook. 
+If you want dbt Cloud to run the job whenever a pull request or commit is made, you can set up continuous integration (CI) for the job. 
 
-To set up a webhook:
+To set up CI:
 
 1. Navigate to the **Triggers** section of the jobs settings page
```
Comment (runleonarun) [inline website/docs/docs/deploy/cloud-ci-job.md]:
```suggestion
1. Create a new job or edit an existing job to open the settings page.
2. Navigate to the **Triggers** section.
```

File: website/docs/docs/deploy/cloud-ci-job.md [hunk 14]
```diff
@@ -22,27 +22,27 @@ If you previously configured your dbt project by providing a generic git URL tha
...
-2. Select **Webhooks**. 
+2. Select **Continuous Integration (CI)**.
```
Comment (runleonarun) [inline website/docs/docs/deploy/cloud-ci-job.md]:
```suggestion
3. Select **Continuous Integration (CI)**. 
```

File: website/docs/docs/deploy/cloud-ci-job.md [hunk 15]
```diff
@@ -22,27 +22,27 @@ If you previously configured your dbt project by providing a generic git URL tha
...
-If you want dbt Cloud to run the job whenever a pull request or commit is made, rather than on a schedule, you can set up a webhook. 
+If you want dbt Cloud to run the job whenever a pull request or commit is made, you can set up continuous integration (CI) for the job. 
 
-To set up a webhook:
+To set up CI:
 
 1. Navigate to the **Triggers** section of the jobs settings page
-2. Select **Webhooks**. 
+2. Select **Continuous Integration (CI)**. 
 3. Select **Run on Pull Requests?** as shown in the following image.
```
Comment (runleonarun) [inline website/docs/docs/deploy/cloud-ci-job.md]:
```suggestion
4. Select **Run on Pull Requests?** as shown in the following image.
```

**Comment by runleonarun** [comment]
@joellabes David updated the UI! I edited the changes and wanted your final ✅  to see if this looks ok to you

**Review by joellabes — APPROVED** [review]
YES wonderful love it. Thank you @thorn14!

**Pull request merged**